antsibull/build_changelog.py

```diff
@@ -596,11 +609,11 @@ def _get_porting_guide_bytes(changelog: Changelog) -> bytes:
                 f"Ansible {version} is based on Ansible-core {base_version}, which is the same"
                 f" major release as Ansible {prev_version}.  Therefore, there is no section on"
                 " ansible-core in this porting guide.  If you are upgrading from Ansible"
-                f" {prev_prev_version}, please first consult the Ansible {prev_version} porting"
+                f" {prev_prev_version}.x, please first consult the Ansible {prev_version} porting"
```

Comment:
@abadger I don't think the `.X` should be here. This is only used for `prev_prev_version >= 3`, and we don't have a porting guide for Ansible 3.X, 4.X, ..., but only for 3, 4, ..., right?

Reply:
Your rationale makes sense.  I'll change it back

Reply:
https://github.com/ansible-community/antsibull/pull/260